geometry_msgs/PoseStamped targetPose
---
bool poseReached